### DISCUSSION TITLE
itemstack blockhandler

### DISCUSSION THREAD

**Discussion by 0alfa** [Minestom/Minestom#1136] [Help] [answered]
how to add a blockhandler to an itemstack?

**Comment by MrlnHi** [comment, marked as answer]
You can't. You put BlockHandlers on Blocks. However, you can put a Tag on the ItemStack that then determines the BlockHandler (say a String tag which, when that String value is used as key to a Map, gives you a BlockHandler) and then get that Tag in the PlayerBlockPlaceEvent
